cloudbuild.yaml

```diff
@@ -1,6 +1,6 @@
 images:
-  - 'gcr.io/$PROJECT_ID/cloud-controller-manager:$_GIT_TAG'
-  - 'gcr.io/$PROJECT_ID/cloud-controller-manager:latest'
+  - 'gcr.io/$PROJECT_ID/k8s-staging-provider-aws:$_GIT_TAG'
+  - 'gcr.io/$PROJECT_ID/k8s-staging-provider-aws:latest'
```

Comment:
both need to change i think to 

'gcr.io/k8s-staging-provider-aws/cloud-controller-manager:latest'

I am not sure what PROJECT_ID is, see https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/cloudbuild.yaml for reference